bluebottle/homepage/views.py

```diff
@@ -9,6 +11,11 @@ class HomePageDetail(generics.GenericAPIView):
     serializer_class = HomePageSerializer
 
     def get(self, request, language='en'):
+
+        # Force requested language
+        translation.activate(language)
```

Comment:
I tried to find a context manager for this, but this url is rather different from others. We specifically ask for a language, that's why I thought it was better to set it here and like this.
What would you suggest @eodolphi ?